### PR TITLE
Fixes #36704 - Reject rerun inputs with null values

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: webpack/JobWizard/autofill.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ export const useAutoFill = ({
             if (typeof rest[key] === 'string') {
               setTemplateValues(prev => ({ ...prev, [input]: rest[key] }));
               setAdvancedValues(prev => ({ ...prev, [input]: rest[key] }));
-            } else {
+            } else if (rest[key].value !== null) {
```
Comment (MariaAga) [inline webpack/JobWizard/autofill.js:79]: is only `null` values are invalid? or should it be `else if (rest[key].value)` ?

Comment (adamruzicka) [reply webpack/JobWizard/autofill.js:79]: I wanted to be as specific as possible, without catching more general falsey values. Not sure if we can get any apart from `false`

**Comment by MariaAga** [comment]
code LGTM, but I cant test it today

**Comment by pondrejk** [comment]
Verified using stream buid + patch. I applied errata from the hosts page, then re-run the respective rex job. No validation errors occurred.

**Review by pondrejk — APPROVED** [review]

**Comment by adamruzicka** [comment]
Thank you @MariaAga & @pondrejk !

**Pull request merged**